website/playground/panels.js

```diff
@@ -150,7 +159,7 @@ export function InputPanel(props) {
       autoCloseBrackets={true}
       matchBrackets={true}
       showCursorWhenSelecting={true}
-      tabWidth={2}
+      tabSize={4}
```

Comment:
http://codemirror.net/doc/manual.html#option_tabSize

Not sure if it should be 2 or 4.